pages/products/index.tsx

```diff
@@ -45,15 +53,23 @@ const Products = () => {
 
     return (
         <Panel>
-            <StatefulTable
+            <Table
                 columns={[
-                    { header: 'Product name', hash: 'name', render: ({ id, name }) => renderName(id, name), sortKey: 'name' },
-                    { header: 'Stock', hash: 'stock', render: ({ stock }) => renderStock(stock), sortKey: 'stock' },
-                    { header: 'Price', hash: 'price', render: ({ price }) => renderPrice(price), sortKey: 'price' },
-                    { header: 'Action', hideHeader: true, hash: 'id', render: ({ id }) => renderAction(id), sortKey: 'id' },
+                    { header: 'Product name', hash: 'name', render: ({ id, name }) => renderName(id, name), isSortable: true },
+                    { header: 'Stock', hash: 'stock', render: ({ stock }) => renderStock(stock), isSortable: true },
+                    { header: 'Price', hash: 'price', render: ({ price }) => renderPrice(price), isSortable: true },
+                    { header: 'Action', hideHeader: true, hash: 'id', render: ({ id }) => renderAction(id), isSortable: true },
                 ]}
                 items={tableItems}
                 itemName="Products"
+                pagination={{
+                    currentPage,
+                    totalItems: meta.pagination?.total,
```

Comment:
since meta itself is a nested prop, we should add optional chaining to pagaination:
```suggestion
                    totalItems: meta?.pagination?.total,
```